src/polycap-photon.c

```diff
@@ -375,6 +376,14 @@ int polycap_photon_launch(polycap_photon *photon, size_t n_energies, double *ene
 		free(photon->scatf);
 		photon->scatf = NULL;
 	}
+//	if (photon->leaks->weight){ //TODO: this will likely not be sufficient if there were multiple occasions of leaks... Perhaps good idea to clear these parameters at the start of this function 
```

Comment:
Geen idee hoe ik dit degelijk invul... Zal iets analoog moeten doen als wat uiteindelijk in `polycap_photon_free()` wordt gedaan. Enkel wss in het begin van de functie uitvoeren ipv op het eind, zodat ik nog weet wat de waarde van `photon->n_leaks` is.